lib/validate_test.go

```diff
@@ -0,0 +1,87 @@
+package samplify
+
+import (
+	"testing"
+)
+
+func TestValidateSurveyURL(t *testing.T) {
+	tables := []struct {
+		id        string
```

Comment:
Chage this to Case name or something and add descriptive messages for each case, so anyone who is looking at this can understand what the case is testing for.